Sources/BraintreeSEPADirectDebit/WebAuthenticationSession.swift

```diff
@@ -12,17 +12,19 @@ class WebAuthenticationSession: NSObject {
     func start(
         url: URL,
         context: ASWebAuthenticationPresentationContextProviding,
-        completion: @escaping (URL?, Error?) -> Void
+        sessionDidDisplay: @escaping (Bool) -> Void,
+        sessionDidComplete: @escaping (URL?, Error?) -> Void
     ) {
-        self.authenticationSession = ASWebAuthenticationSession(
+        let authenticationSession = ASWebAuthenticationSession(
```

Comment:
shouldnt `var authenticationSession` be deleted then?

Reply:
yes. I was testing out persisting reference in the class to see if I can get the system notification on `applicationDidBecomeActive` on browser launch but it didn't work.